src/util.ts

```diff
@@ -214,3 +214,18 @@ export const getConversionRateLift = (
 	const personalizedRate = getConversionRate( personalized.conversions, personalized.views );
 	return getLift( personalizedRate, fallbackRate );
 };
+
+/**
+ * Track a user action/event to Segment
+ *
+ * @param module Module where the action happens.
+ * @param action Action name.
+ * @param parameters Additional details/parameters of the action.
```

Comment:
Check your doc block syntax - this has no indication of the parameter data types